rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -39,6 +39,10 @@ class OneFrameProcessor : public FrameProcessor {
   }
 
   void processFrame(std::unique_ptr<folly::IOBuf> buf) override {
+    if(!transport_) {
```

Comment:
Might we be silencing an issue in here? Why the transport_ object might be null and we might be still calling processFrame on the OneFrameProcessor object?